setup.py

```diff
@@ -10,6 +10,7 @@
 import shutil
 
 SETUP_DIR = os.path.dirname(os.path.abspath(__file__))
+REQUIREMENTS = ["guacamol", "matplotlib", "torch", "joblib", "numpy", "tqdm", "cython", "nltk", "flake8"]
 
```

Comment:
Do we really need flake8 for the install? Maybe we can remove it since I assume it's only for dev